lambda-api/api_lambda_service.proto

```diff
@@ -1,2 +1,37 @@
 syntax = "proto3";
-package api_lambda_service;
\ No newline at end of file
+package api_lambda_service;
```

Comment:
Incorporating some feedback - let's make this package `kurtosis_lambda_api_service`

Reply:
1) this adds `kurtosis` to the package name, whcih was a point of confusion during developer feedback session
2) flips the ordering of "api_lambda" -> "lambda_api" which is a bit easier to read in English